lib/sufia/redis_event_store.rb

```diff
@@ -1,5 +1,29 @@
 module Sufia
   class RedisEventStore
+    class << self
+      def for(key)
+        new(key)
+      end
+
+      # @return [Fixnum] the id of the event
+      def create(action, timestamp)
+        event_id = Redis.current.incr("events:latest_id")
+        Redis.current.hmset("events:#{event_id}", "action", action, "timestamp", timestamp)
+        event_id
+      rescue Redis::CommandError => e
+        logger.error("unable to create event: #{e}")
+        nil
+      end
+
+      def logger
+        @logger = (defined?(Rails) && Rails.logger) || CurationConcerns::NullLogger.new
```

Comment:
I didn't know the context in which this class was used. For example, if it was used in Resque, but I guess that has Rails as well? Either way, I'm :+1: to removing it.